node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/identity/AthenzCredentialsMaintainer.java

```diff
@@ -145,6 +160,21 @@ public void clearCredentials(NodeAgentContext context) {
         lastRefreshAttempt.remove(context.containerName());
     }
 
+    @Override
+    public Duration certificateExpiry(NodeAgentContext context) {
```

Comment:
Consider improving name. `certificateExpiry` sounds like a specific moment in time, but looks like this returns time left until expiry? `certificateLifetime`?